tests/rt.conf

```diff
@@ -2,35 +2,36 @@
 # S2S tests                                                                                                                                                                    #
 ###################################################################################################################################################################################
 
-COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1,FV3_GFS_v16_coupled_p7_rrtmgp,FV3_GFS_v16_coupled_p8      | - wcoss_cray                            | fv3 |
+COMPILE | -DAPP=S2SW -DUFS_GOCART=ON -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1,FV3_GFS_v16_coupled_p7_rrtmgp,FV3_GFS_v17_coupled_p8b     | - wcoss_cray                            | fv3 |
```

Comment:
Can we call this FV3_GFS_v17_coupled_p8? And update this suite file for future P8 prototypes?

Reply:
I'm following the naming convention from @yangfanglin, so I'll let him weigh in on if he would be okay with this change or not or if there's a need for being able to run the multiple options.

Reply:
it is fine to call it FV3_GFS_v17_coupled_p8